src/docs/product/issues/issue-details/performance-issues/uncompressed-asset/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Uncompressed Asset"
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/uncompressed-asset/
+description: "Learn more about Uncompressed Asset issues and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+Uncompressed asset issues are when one of the files your users are downloading to load your browser page was not compressed when transferred. This likely indicates a misconfiguration of the server or CDN (content delivery network) serving the asset file.
+
```

Comment:
```suggestion
Uncompressed asset issues happen when a file that needs to be downloaded in order to load a browser page, doesn’t get compressed while being transferred. This may indicate a misconfiguration of the server or CDN (content delivery network) that’s serving the asset file.

```